src/planner/ostree.rs

```diff
@@ -183,6 +194,16 @@ impl Planner for Ostree {
                 .map_err(PlannerError::Action)?
                 .boxed(),
         );
+
+        if Path::new("/etc/fish").is_directory() {
+            plan.push(
+                CreateDirectory::plan(&OSTREE_FISH_PROFILE_LOCATION, None, None, 0o0755, true)
```

Comment:
Also, is 0o755 the correct mode for this? (I genuinely don't know)

Reply:
I don't know either :)

Reply:
755 is right